tests/test_compile.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+import re2
+
+class TestCompile(unittest.TestCase):
+    def test_raise(self):
+        with self.assertRaisesRegexp(
+            re2.error,
+            'no argument for repetition operator: *'
```

Comment:
Should the `*` be escaped because this is a regex?

Reply:
that's ironic -- the code that led me to write this pull request was `re.compile('*')` and here I am abusing `*` :)